src/services/codefixes/helpers.ts

```diff
@@ -394,11 +394,16 @@ namespace ts.codefix {
             }
         }
         const maxNonRestArgs = maxArgsSignature.parameters.length - (signatureHasRestParameter(maxArgsSignature) ? 1 : 0);
-        const maxArgsParameterSymbolNames = maxArgsSignature.parameters.map(symbol => symbol.name);
-        const parameters = createDummyParameters(maxNonRestArgs, maxArgsParameterSymbolNames, /* types */ undefined, minArgumentCount, /*inJs*/ false);
+        const maxArgsParameterSymbolNames = [];
+        const maxArgsParameterTypes = [];
+        for (const parameter of maxArgsSignature.parameters) {
+            maxArgsParameterSymbolNames.push(parameter.name);
+            maxArgsParameterTypes.push(factory.createKeywordTypeNode(SyntaxKind.UnknownKeyword));
+        }
+        const parameters = createDummyParameters(maxNonRestArgs, maxArgsParameterSymbolNames, maxArgsParameterTypes, minArgumentCount, /*inJs*/ false);

```

Comment:
what happens if you change `createDummyParameters` to produce `unknown` instead of `any` too? This is probably a change that all the codefixes would benefit from.